tox.ini

```diff
@@ -48,16 +48,16 @@ commands =
    pre-commit run [] --all-files --show-diff-on-failure
 
 [testenv:vendoring]
+basepython = python3.8
 skip_install = True
 commands_pre =
 deps =
-    invoke
-    requests
+    vendoring == 0.2.2
```

Comment:
nitpicking, maybe `vendoring==0.2.2` to match the format below?

Reply:
If there's other changes requested, and I remember to change this, I'll do it. :)